src/pages/[platform]/build-a-backend/data/enable-logging/index.mdx

```diff
@@ -0,0 +1,95 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Enable logging for your data resource',
+  description: 'Enable logging for your data resource',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ]
+};
+
+export function getStaticPaths() {
+  return getCustomStaticPath(meta.platforms);
+}
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+You can enable logging to debug your GraphQL API using CloudWatch logs.
+
+## Default Logging
+
+Default logging can be enabled by setting the `logging` property to `true` in the call to `defineData`. For example:
+
+```ts title="amplify/data/resource.ts"
+export const data = defineData({
+  some_schema,
+  some_authorizationModes,
+  logging: true
+});
+```
+
+Using `logging: true` applies the default configuration:
+- `excludeVerboseContent: true`
+- `fieldLogLevel: 'none'`
+- `retention: '1 week'`
+
+## Custom Logging
+
+You can override individual fields by providing a `DataLogConfig` object. For example:
```

Comment:
```suggestion
You can customize individual configuration values by providing a [`DataLogConfig`](#datalogconfig-fields) object. For example:
```

nit for the term "override" since we typically reserve this for overriding resource properties via CDK